src/responses/citra.json

```
@@ -5,6 +5,7 @@
       "requirements": { "reply": "Please refer to our **Frequently Asked Questions**. The only requirements for Citra are a GPU that supports at least OpenGL 4.3 and a 64-bit OS, but you definitely want a processor with the highest possible performance per core. <https://citra-emu.org/wiki/faq/>"},
       "roms": { "reply": "Please read our __community rules__. Warez/downloading games talk is strictly prohibited. To prevent legal issues, you are not allowed to post links or refer to any kind of ROM, NAND, ISO, game, or other copyrighted material that has been illegally obtained or shared. <https://citra-emu.org/rules/>"},
       "dump-game": { "reply": "Please refer to our __game dumping guides__. \nFor Cartridges: <https://citra-emu.org/wiki/dumping-game-cartridges/> \nFor Installed Titles: <https://citra-emu.org/wiki/dumping-installed-titles/> \nTo dump DLC and Updates, use this guide <https://citra-emu.org/wiki/dumping-updates-and-dlcs/> and install the results through `File -> Install CIA...`"},
+      "dumping": {"reply": "Dumping is the act of copying a game from your system/gamecart to an SD card. \nDumping is legal, however sharing the dumps with anyone else is piracy and thus is illegal."},
```

```suggestion
      "dumping": {"reply": "Dumping is the act of creating an archival copy of your own legally purchased games. This process requires a hacked console to perform the dump \nDumping is legal, however sharing your dumped archival copies with anyone else is illegal."},
```
Clarifies on what dumping does, adds section about needing a console, clarifies what "dump" is when talking about the legality.

---

Will change it later, I did originally have it set to something like that, but must have changed it unintentionally.